server/routes/upload.js

```diff
@@ -25,11 +25,6 @@ router.post('/',
       res.status(400).render('upload', { user: req.user._json, error: 'File must be a PDF' });
       return;
     }
-    // verify user isn't very unoriginal
-    if (req.file.originalname.toLowerCase() === 'resume.pdf') {
-      res.status(400).render('upload', { user: req.user._json, error: 'Really? "resume.pdf"? C\'mon.' });
-      return;
-    }
```

Comment:
It would be a good idea to recommend to uploaders that they use a file name that represents who it is, e.g. name_resume.pdf, as an aid to recruiters (and people doing referrals), but I agree with not erroring.

Reply:
The recommendation that people change their filename is I think generally good and should be kept. I'm even in favor of it erroring personally.

Reply:
In that case, how about a useful error message instead of a snarky one?

Reply:
Yeah, the snark isn't really productive. Make it a suggestion, at the minimum. (github respond be email really doesn't handle threads :( )